assets/src/blocks/Counter/CounterEditor.js

```diff
@@ -0,0 +1,130 @@
+import { Component, Fragment } from '@wordpress/element';
+import { InspectorControls } from '@wordpress/block-editor';
+import {
+  TextControl,
+  TextareaControl,
+  PanelBody
+} from '@wordpress/components';
+
+import { URLInput } from "../../components/URLInput/URLInput";
+
+import { CounterFrontend } from './CounterFrontend';
+
+const { RichText } = wp.editor;
+const { __ } = wp.i18n;
+
+export class CounterEditor extends Component {
+  constructor(props) {
+    super(props);
+
+    this.toAttribute = this.toAttribute.bind(this);
+  }
+
+  toAttribute(attributeName) {
+    const { setAttributes } = this.props;
+    return value => {
+      setAttributes({ [attributeName]: value });
+    }
+  }
+
+  renderEdit() {
+    const { attributes } = this.props;
+
+    return (
+      <Fragment>
+        <InspectorControls>
+          <PanelBody title={__('Setting', 'p4ge')}>
```

Comment:
The p4ge is the leftover from very initial gutenberg demo(`planet4-gutenberg-experiment` - p4ge), the proper values would be the `planet4-blocks-backend` and `planet4-blocks`

Reply:
Thank you for the clarification! I'll change it 🙂

Reply:
`planet4-blocks-backend` is to be used in the editor?

Reply:
yes , that's right.